sql/core/src/test/resources/sql-tests/results/postgreSQL/strings.sql.out

```diff
@@ -907,7 +907,7 @@ SELECT rpad('hi', 5, '')
 -- !query 110 schema
 struct<rpad(hi, 5, ):string>
 -- !query 110 output
```

Comment:
Can you leave some comments in the corresponding query (`inputs/postgreSQL/strings.sql`) about why the result is different from PostgreSQL one?

Reply:
ok i will add comment

Reply:
-- PostgreSQL return: hi:  because not implemented as per the rpad behaviours(In case of empty pad string, the return value is null) , @maropu , this is ok ?